### PR TITLE
refactor(cli): cleanup info command

### PR DESCRIPTION
**Pull request by amrbashir**
### What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [x] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [x] No

### Checklist
- [ ] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [ ] I have added a convincing reason for adding this feature, if necessary

### CODE REVIEW

File: tooling/cli/src/info/packages_rust.rs [hunk 1]
```diff
@@ -211,26 +211,16 @@ pub fn items(app_dir: Option<&PathBuf>, tauri_dir: Option<PathBuf>) -> Vec<Secti
...
-              Status::Neutral,
-            ))
-          },
-          || None,
-          false,
-        );
+        let item = SectionItem::new().description(format!(
+          "{} {}: {}{}",
+          dep,
+          "🦀",
```
Comment (lucasfernog) [inline tooling/cli/src/info/packages_rust.rs]: I would avoid using this.. I only see a question mark :D

Comment (amrbashir) [reply tooling/cli/src/info/packages_rust.rs]: lol this is just an emoji, I was more concerned about the Node.js one, I will revert theses changes anyways, I will add them later if I find a good way to have a fallback

File: tooling/cli/src/info/packages_nodejs.rs [hunk 2]
```diff
@@ -238,46 +238,40 @@ pub fn items(
...
-                if !(version.is_empty() || latest_ver.is_empty()) {
-                  let version = semver::Version::parse(version.as_str()).unwrap();
-                  let target_version = semver::Version::parse(latest_ver.as_str()).unwrap();
+        if version.is_empty() {
+          format!("{} {}: not installed!", package, "".green())
+        } else {
+          format!(
+            "{} {}: {}{}",
+            package,
+            "".green(),
```
Comment (amrbashir) [inline tooling/cli/src/info/packages_nodejs.rs]:
```suggestion
            "[NPM]".dimmed(),
```

File: tooling/cli/src/info/packages_rust.rs (hunk 1 above)
Comment (amrbashir) [inline tooling/cli/src/info/packages_rust.rs]:
```suggestion
          "[RUST]".dimmed(),
```

File: tooling/cli/src/info/packages_rust.rs [hunk 3]
```diff
@@ -212,96 +212,58 @@ pub fn items(app_dir: Option<&PathBuf>, tauri_dir: Option<PathBuf>) -> Vec<Secti
...
+  let tauri_cli_rust_item = SectionItem::new().action(|| {
+    std::process::Command::new("cargo")
+      .arg("tauri")
+      .arg("-V")
+      .output()
+      .ok()
+      .map(|o| {
+        if o.status.success() {
+          let out = String::from_utf8_lossy(o.stdout.as_slice());
+          let (package, version) = out.split_once(" ").unwrap_or_default();
```
Comment (martinfrances107) [inline tooling/cli/src/info/packages_rust.rs]:
```suggestion
          let (package, version) = out.split_once(' ').unwrap_or_default();
```

Comment (martinfrances107) [reply tooling/cli/src/info/packages_rust.rs]: fixes a clippy warning

File: tooling/cli/src/info/packages_rust.rs [hunk 4]
```diff
@@ -212,96 +212,58 @@ pub fn items(app_dir: Option<&PathBuf>, tauri_dir: Option<PathBuf>) -> Vec<Secti
...
+          let latest_ver = crate_latest_version(package).unwrap_or_default();
+          format!(
+            "{} {}: {}{}",
+            package,
+            "[RUST]".dimmed(),
+            version.split_once("\n").unwrap_or_default().0,
```
Comment (martinfrances107) [inline tooling/cli/src/info/packages_rust.rs]:
```suggestion
            version.split_once('\n').unwrap_or_default().0,
```

Comment (martinfrances107) [reply tooling/cli/src/info/packages_rust.rs]: second clippy fix

**Review by martinfrances107 — CHANGES_REQUESTED** [review]

**Review by lucasfernog — APPROVED** [review]

**Pull request merged**